--- wake.cpp.1~	2010-08-06 17:46:22.000000000 -0400
+++ wake.cpp	2019-10-15 13:52:38.638469633 -0400
@@ -24,14 +24,14 @@
 	// x and z were declared as "long" in Wheeler's paper, which is a signed type. I don't know if that was intentional, but it's too late to change it now. -- Wei 7/4/2010
 	CRYPTOPP_COMPILE_ASSERT(sizeof(x) == 4);
 	static int tt[10]= {
-		0x726a8f3b,								 // table
-		0xe69a3b5c,
-		0xd3c71fe5,
-		0xab3c73d2,
-		0x4d3a8eb3,
-		0x0396d6e8,
-		0x3d4c2f7a,
-		0x9ee27cf3, } ;
+		(int) 0x726a8f3b,								 // table
+		(int) 0xe69a3b5c,
+		(int) 0xd3c71fe5,
+		(int) 0xab3c73d2,
+		(int) 0x4d3a8eb3,
+		(int) 0x0396d6e8,
+		(int) 0x3d4c2f7a,
+		(int) 0x9ee27cf3, } ;
 	t[0] = k0;
 	t[1] = k1;
 	t[2] = k2;
